src/en/administration/install_docker.md

```diff
@@ -8,8 +8,8 @@ mkdir /lemmy
 cd /lemmy
 
 # download default config files
-wget https://raw.githubusercontent.com/LemmyNet/lemmy/main/docker/prod/docker-compose.yml
-wget https://raw.githubusercontent.com/LemmyNet/lemmy/main/docker/lemmy.hjson
+wget https://raw.githubusercontent.com/LemmyNet/lemmy-ansible/main/templates/docker-compose.yml
```

Comment:
That won't work, because the template file requires a lot of manual editing for version numbers and such: https://github.com/LemmyNet/lemmy-ansible/blob/main/templates/docker-compose.yml

Reply:
Hmm true. Then we have one more file to keep up to date (like pictrs versions).